src/static/bitbucket/components/App.tsx

```diff
@@ -54,6 +54,11 @@ const App = () => {
   const [queue, setQueue] = useState<QueueResponse['queue'] | undefined>();
   const [_, setLoadStatus, loadStatusRef] = useState<LoadStatus>('not-loaded');
   const [state, dispatch] = useState(initialState);
+  const [isChecked, setIsChecked] = useState(true);
```

Comment:
Can this naming be more representative? `isSquashMergeChecked` perhaps? 
Or consider using an enum `mergeStrategy` here.